examples/mobile/UIComponents/components/dialogs/dialog.js

```diff
@@ -0,0 +1,13 @@
+(function () {
+	var page = document.getElementById("demo-dialog-page"),
+		buttons = document.querySelectorAll(".ui-btn"),
+		idx;
+
+	page.addEventListener("pageshow", function () {
```

Comment:
You have to take account remove this listener...
Now, this listener will be add many times.

You can do similar like `examples/mobile/UIComponents/components/content/searchinput/searchinput.js`

Reply:
I remove addEventListener.
and only call tau.history.back() in for loop.

done.